front/tests/chat/data/chat-eval-inputs.json

```diff
@@ -0,0 +1,99 @@
+[
+  {
+    "question": "What's the address of the offsite and the door code?",
+    "rules": [
+      "contains the address 5 rue Fromentin",
+      "contains the door code 36A12"
+    ],
+    "time": "2023-06-27 10:00",
+    "params": { "sources": ["notion"] }
+  },
+  {
+    "question": "What is Daphné working on?",
+    "rules": [
+      "mentions that she works on Events",
+      "mentions that she is a runner this week"
```

Comment:
I was about to say this won't be true next week, then I read [this](https://www.notion.so/dust-tt/Design-Doc-Chat-Eval-0b849d4c66564f1e9c0c31d537c96f78?pvs=4#3aa452ea067a4dacbd19887d192aab8c) on your Design doc and 🤩